.travis.yml

```diff
@@ -56,44 +45,9 @@ jobs:
     - php: nightly
     - env: DB=pgsql POSTGRESQL_VERSION=11.0
 
-  exclude:
-    - php: 7.1
-      env: DB=sqlite
-    - php: 7.1
-      env: DB=mysql
-    - php: 7.1
-      env: DB=mysqli
-    - php: 7.2
-      env: DB=sqlite
-    - php: 7.2
-      env: DB=mysql
-    - php: 7.2
-      env: DB=mysqli
-    - php: 7.3
-      env: DB=mysql
-    - php: 7.3
-      env: DB=mysqli
-    - php: nightly
-      env: DB=mysql
-    - php: nightly
-      env: DB=mysqli
-
   include:
-    - stage: Test
-      php: 7.1
-      env: DB=sqlite
-
-    - stage: Test
-      php: 7.2
-      env: DB=sqlite COVERAGE=yes
-
-    - stage: Test
-      php: 7.2
-      env: DB=mysql COVERAGE=yes
-
-    - stage: Test
-      php: 7.2
-      env: DB=mysqli COVERAGE=yes
+    # The matrix below has been generated with:
+    # https://gist.github.com/BenMorel/67e5d94c864af9b9ae1d3a1db8010b98
```

Comment:
Two questions:

- should I leave the start and end comments around the auto-generated code?
- how will people, without a pointer to the gist, commit or PR, will know that an automated script was used to output these jobs, and that they should maybe attempt to use it rather than modifying the matrix manually? Of course they can see the commit history for the `.travis.yml` file, but who does this when modifying a file?

Reply:
> should I leave the start and end comments around the auto-generated code?

Yes, please.

> how will people, without a pointer to the gist, commit or PR, will know that an automated script was used to output these jobs […]

It's not always needed. E.g. if we want to make any trivial changes to the matrix (e.g. remove PHP 7.1 or switch code coverage from PHP 7.2 to PHP 7.3), it can be easily done without the script. For non-trivial changes, first you'd have to rework the script. It already did it job by building the current well-structured matrix.

Reply:
So I removed the link to the script, and left the comments, but it feels weird:

> The matrix below has been auto-generated

Auto-generated with *what*? It does not make sense to keep it this way, IMHO. Either we keep the comment and link to the script (or to this PR), or we remove the comment altogether.

What do you think?

Reply:
This comment should go away too together with "end of auto-generated code".

Reply:
Looks like we did not understand each other above. Done.